src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabPersonalAccessTokenCredential.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import org.jenkinsci.test.acceptance.plugins.credentials.BaseStandardCredentials;
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.jenkinsci.test.acceptance.po.PageObject;
+
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+@Describable("GitLab Personal Access Token")
+public class GitLabPersonalAccessTokenCredential extends BaseStandardCredentials {
+
+    public GitLabPersonalAccessTokenCredential(PageObject context, String path) {
+        super(context, path);
+    }
+
+    public GitLabPersonalAccessTokenCredential(PageAreaImpl area, String relativePath) {
+        super(area, relativePath);
+    }
+
+    public void setToken(String token) {
+        control(by.path("/credentials/token")).set(token);
```

Comment:
I don't quite understand this, I used a lot `control` should I refactor everything? :( 
And then, the next comment suggests I shoud use `control` instead of `find`? https://github.com/jenkinsci/acceptance-test-harness/pull/1335/files#r1314662604

Reply:
> I don't quite understand this, I

```
class ... {
  private Control token = control(by.path("/credentials/token"));
  
  //...,
  public void setToken(String gitLabToken) {
     token.set(gitLabToken);
   }
   
}